### PR TITLE
Escape HTML chars in PR titles for merge queue page

### PR DESCRIPTION
**Pull request by dobbymoodge**

### CODE REVIEW

File: test_pull_requests [hunk 1]
```diff
@@ -15,12 +15,12 @@
 
 require 'rubygems'
 require 'uri'
+require 'cgi'
```
Comment (dobbymoodge) [inline test_pull_requests]: Moved this because `hub` aliases the `CGI` module with one that lacks `escapeHTML`

**Comment by dobbymoodge** [comment]
@danmcp @stevekuznetsov

**Comment by danmcp** [comment]
@dobbymoodge Been tested?

**Comment by dobbymoodge** [comment]
@danmcp and it's properly displaying the title for this PR: https!//github.com/openshift/origin/pull/12605

File: test_pull_requests (hunk 1 above)
Comment (stevekuznetsov) [inline test_pull_requests]: They love their weird aliases ...

**Pull request merged**